src/Process/TreeLayerProcessing.js

```diff
@@ -0,0 +1,10 @@
+export default function updateTreeLayer(processNodeFn, context, layer, node) {
+    if (!node) {
```

Comment:
The framework may have the responsibility of `initRootNodes` when creating the layer